Autopilot/PathManager/Inc/pathDatatypes.hpp

```diff
@@ -0,0 +1,43 @@
+#ifndef PATH_DATATYPES_HPP
+#define PATH_DATATYPES_HPP
+
+/********************************************************************
+* Enums
+********************************************************************/
+
+enum _ModifyFlightPathCommand { NO_FLIGHT_PATH_EDIT = 0, INITIALIZE_FLIGHT_PATH, APPEND, INSERT, UPDATE, DELETE, NUKE }; // Used by cruisingState
+enum _GetNextDirectionsCommand { REGULAR_PATH_FOLLOWING = 0, TOGGLE_HOLDING, TOGGLE_HEAD_HOME }; // Used by cruisingState
+
+/********************************************************************
+* Imporatant Structs
+********************************************************************/
+
+struct Telemetry_Waypoint_Data_t {
+    long double latitude;
+    long double longitude;
+    int altitude;
+    float turnRadius;
+    int waypointType; // 0 = Path follow, 1 = Orbit, 2 = Hold
+};
+
+struct Telemetry_PIGO_t {
+    /* Parameters for the waypoint manager (crusingState) */
+    int numWaypoints;
+    
+    _ModifyFlightPathCommand waypointModifyFlightPathCommand; 
+    bool initializingHomeBase; // 0 = no, 1 = yes
+    _GetNextDirectionsCommand waypointNextDirectionsCommand; 
+    int holdingAltitude;
+    int holdingTurnRadius;
```

Comment:
Can we have these types match up exactly with the document? It'l prevent some confusion when GOrdon encodes/decodes the mavlink struct

Reply:
I'll update the docs sometime tomorrow

Reply:
docs are fine. I meant like change int to uint8_t or whatever the doc specifies

Reply:
Oh are you referring to the enums? I am keeping them there for readability when this struct is used later. Gordon can initialize the parameters by checking if the value is valid and casting the int to the enum